### ISSUE TITLE
No need to prompt "Which variants you would like to publish?" when only 1 variant has changes 

### ISSUE THREAD

**Issue by martinbrud** [umbraco/Umbraco-CMS#9480]
I love the possibility to use language variants, but one thing bothers me: Every time I hit "Publush" I must react on this message: "Which variants you would like to publish?"

See screenshot: https://i.imgur.com/SbBiJgj.png

Most times I have only made changes to one variant and therefore there is only that variant one to consider. An therefore I would only have hit the Publish button if I meant to publish it. 

Desired solution: Publish right away without a confirm box if there is only made changes to one variant.

**Comment by nul800sebastiaan** [comment]
Hmm, I can't reproduce this in version 8.8.2 @martinbrud. It sounds familiar but I can't find an issue or pull request for fixing this. I am going to assume for now that you're on an older version before this was fixed, if you're at the latest version (8.9.1) then let us know and we can have another look.

**Comment by martinbrud** [comment]
Hi Sebastiaan,

Thank you for your message.

I *am *experiencing it on v. 8.9.1:
[image: image.png]

I looks ok when I have made changes to more than one language variant:
[image: Both edited.gif]

But when I have made changes to only one language version it also asks me
to confirm that I mean to publish that one. And that is not necessary since
that why I hit "Publish" :)
[image: Only one.gif]



Bedste hilsner / best regards,
Martin

--
*Martin Rud - bedre digitale kundeoplevelser / better digital customer
experiences*
Freelance webudvikler og digital rådgiver / Freelance web developer and
digital adviser
T +45 4128 4388
www.martinrud.dk

*Jeg udvikler websites og rådgiver om virksomheders digitale
tilstedeværelse. Hvor mine kompetencer ophører, har jeg et netværk af
udviklere, designere, tekstforfattere, strateger m.fl., som jeg trækker
ind, så projekterne bliver løst af folk med de rigtige kompetencer.*
